### PR TITLE
Remove Reqs Crates Price Defines, and outdated crates

### PR DESCRIPTION
**Pull request by Huffie56**
## About The Pull Request
1-Remove the useless intermediary that was the define RO_PRICE for just numbers.
2-Cleaned up useless crate and fuse some other together.

## Why It's Good For The Game
1-Removing define will make the code cleaner and simpler.
2-less trap for and useless options.

## Changelog

:cl:
add: fuse multiple crates into a new one for large pouch "Large Pouch 2x(pistol,magazine,general)"
add: fuse multiple crates into a new one for surgery "surgery crate(tray,anesthetic,surgeon gear)"
removal: remove M3 Pattern Medium Armor Crate (x5 helmet, x5 armor) and M3 Pattern Light Armor Crate (x3) crates.
removal: remove M3 Pattern Heavy Armor Crate (x3) and rail attachments crate (x2 each) crates.
removal: remove muzzle attachments crate (x2 each) and bayonet attachment crate (x8) crates.
removal: remove underbarrel attachments crate (x2 each) and general pouches crate (2x normal, 1x medium, 1x large) crates.
removal: remove weapons pouches crate (1x bayonet, pistol, explosive) and survival pouches crate (1x radio, flare, survival) crates.
removal: remove construction pouches crate (1x electronics, tools, construction) and Lightweight IMP Backpack Crate (x4) crates.
removal: remove Marine Satchel Crate (x4) and M39 Belt Holster Crate (x2) and M44 Belt Holster Crate (x2) crates.
removal: remove M4A3 Belt Holster Crate (x2) and marine outfit crate and anesthetic crate (x4 masks, x4 tanks) crates.
removal: remove sterile equipment surplus sidearms and surplus shotguns and M39 SMG and M41A MK2 Crate crates.
removal: remove old define RO_PRICE and directly input the cost variable number for crates.
/:cl:

### CODE REVIEW

**Comment by Huffie56** [comment]
Found this PR that could conflict so i will keep old define until for now so any PR made on it still work
https://github.com/cmss13-devs/cmss13/pull/1461

**Comment by Huffie56** [comment]
I am keeping the double define system until other PR get merge before this don't want to create any conflicts of PR.
also i am not touching multiple category since i saw other PR including mine sometime on them (misc, operation, hardpoint_module)

**Comment by Huffie56** [comment]
Seem to work on my end
image

**Comment by fira** [comment]
If we're gonna define them as such, we might as well remove the define entirely

**Comment by Huffie56** [comment]
> If we're gonna define them as such, we might as well remove the define entirely

I was thinking about it but i didn't saw where the price is calculated so i put the idea aside for now.
I can keep working on it....
but yea could remove define since changing it didn't seem to be link to anything ...

File: code/datums/supply_packs/_supply_packs.dm
Comment (fira) [inline code/datums/supply_packs/_supply_packs.dm:38]: Should be 40 here then!

**Comment by fira** [comment]
But yeah i'm all for removing these placeholder defines that really don't serve any purpose other than being embarassing and confusing. There's a number of gun defines i'd also be all for shooting down personally...

File: code/datums/supply_packs/_supply_packs.dm
Comment (Huffie56) [reply code/datums/supply_packs/_supply_packs.dm:38]: i put a seven because it's the lowest number we should imput for cost.

**Comment by fira** [comment]
Well it's fine by me as is, it's been time we shoot down these defines, i'd just want someone else to review this too since it's really a lot of a subjective change

**Comment by Huffie56** [comment]
> Well it's fine by me as is, it's been time we shoot down these defines, i'd just want someone else to review this too since it's really a lot of a subjective change

thanks for reviewing my PR's can you take a look at the other PR i made some are very short?
would be much appreciated.

File: code/datums/supply_packs/clothing.dm
Comment (fira) [inline code/datums/supply_packs/clothing.dm:42]: should keep this, autoinjector pouches are limited

File: code/datums/supply_packs/clothing.dm
Comment (fira) [inline code/datums/supply_packs/clothing.dm:29]: should keep this, large magazine pouches are limited

File: code/datums/supply_packs/clothing.dm
Comment (fira) [inline code/datums/supply_packs/clothing.dm:237]: i think we should keep this one too, it has shenanigans potential

File: code/datums/supply_packs/medical.dm
Comment (fira) [inline code/datums/supply_packs/medical.dm:6]: this one has legit usecases, marines often ask for medical supplies to be dropped. it's more efficient to get belts from medical of course buuuuuuuuut...

File: code/datums/supply_packs/medical.dm
Comment (fira) [inline code/datums/supply_packs/medical.dm:31]: same

File: code/datums/supply_packs/medical.dm
Comment (fira) [inline code/datums/supply_packs/medical.dm:27]: could use a more descriptive name, like "tray, anesthetic" or something rather than "all the items"

**Review by fira — CHANGES_REQUESTED** [review]
Some extra notes concerning the changes to crates, there's a few that should be realistically kept

I'll change MR name to something more descriptive

File: code/datums/supply_packs/medical.dm
Comment (Huffie56) [reply code/datums/supply_packs/medical.dm:6]: didn't know that never saw it being used i can leave it...

File: code/datums/supply_packs/medical.dm
Comment (Huffie56) [reply code/datums/supply_packs/medical.dm:31]: ok

File: code/datums/supply_packs/medical.dm
Comment (Huffie56) [reply code/datums/supply_packs/medical.dm:27]: done

File: code/datums/supply_packs/armor.dm
Comment (Huffie56) [reply code/datums/supply_packs/armor.dm:60]: it's not deleted just moved

File: code/datums/supply_packs/clothing.dm
Comment (Huffie56) [reply code/datums/supply_packs/clothing.dm:42]: done

**Comment by Huffie56** [comment]
> Some extra notes concerning the changes to crates, there's a few that should be realistically kept
> 
> I'll change MR name to something more descriptive

Done waiting for review.

File: code/datums/supply_packs/clothing.dm
Comment (Huffie56) [reply code/datums/supply_packs/clothing.dm:237]: ok

File: code/datums/supply_packs/clothing.dm
Comment (Huffie56) [reply code/datums/supply_packs/clothing.dm:29]: ok created a new crate with the large mag pouche

**Comment by fira** [comment]
Sorry, seems other changes to crates are conflicting with this

You can hit Resolve Conflict at bottom, and GitHub will show you both changes made, then you modify it into the final change you want

**Comment by Huffie56** [comment]
> Sorry, seems other changes to crates are conflicting with this
> 
> You can hit Resolve Conflict at bottom, and GitHub will show you both changes made, then you modify it into the final change you want

no problem i fix it and some conflicts where from my own merger so.... just happy to see my PR move along..

File: code/datums/supply_packs/clothing.dm
Comment (Huffie56) [reply code/datums/supply_packs/clothing.dm:29]: done

Comment (Huffie56) [reply code/datums/supply_packs/clothing.dm:29]: it's fine for me as long as your comment are productive... i corrected it.

**Review by fira — APPROVED** [review]

File: code/datums/supply_packs/_supply_packs.dm
Comment (harryob) [inline code/datums/supply_packs/_supply_packs.dm:44]:
```suggestion
	var/contraband = 0			/// Tell if the crate is a contraband value 1 or not 0.
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]

**Review by fira — APPROVED** [review]

File: code/datums/supply_packs/ammo.dm
Comment (ThePiachu) [inline code/datums/supply_packs/ammo.dm:475]: Contraband

**Review by ThePiachu — CHANGES_REQUESTED** [review]

File: code/datums/supply_packs/ammo.dm
Comment (ThePiachu) [inline code/datums/supply_packs/ammo.dm:475]:
```suggestion
//------------------------Contraband----------------
```

**Review by ThePiachu — APPROVED** [review]

**Pull request merged**
